.taskcluster.yml

```diff
@@ -5,9 +5,8 @@ policy:
     pullRequests: public
 tasks:
     - $let:
-          repository:
-              name: tools
-              description: Android L10n Tooling
+          name: tools
```

Comment:
I'm *hoping* we don't overwrite `name`... https://hg.mozilla.org/ci/ci-configuration/file/33c8922af2808175c12199a0f3ec6c30a26e00cd/build-decision/src/build_decision/cron/schema.yml#l12 or anywhere else where we don't namespace our inputs.

Reply:
I kind of don't like these keys anyway... I'll just put them directly in the env.

Reply:
I do think we're going the wrong direction: ideally we'd have a simple set of variables to populate at the top of this repo, tightly namespaced, and we'd be able to standardize the body of the `.tc.yml` in preparation of moving the body to a template somewhere else. It seems like we're moving towards having to understand and edit the body of the `.tc.yml` which will likely lead to missing things that need to change in future repos.

Reply:
Hmm, I get that point of view. Until we actually have a template though, it seems like trying to templatize things just makes it more complicated in the meantime. My preference would be to only start moving in that direction once doing so actually simplifies.. but it's not a sticking point or anything.

Reply:
Please feel free to r- if you'd like me to go back :)

Reply:
I tend to never ever want to have to look at the body of the .tc.yml unless something is horribly broken; this guides my opinion... I'm hoping we can find that `n` repos all have pretty much the same .tc.yml except for a block at the top, we standardize on something that works for all of them, then we have the template essentially defined for that set of repos.

It seems like you and several other releng are ok with .tc.yml hacking, however, so if you have a different roadmap to get us to a template solution, then I don't want to block this fix.

Reply:
> Please feel free to r- if you'd like me to go back :)

I kind of hate this :(

Do you have strong objections to, say, reviving the `taskgraph.repository.*` ? Could also be `taskgraph_vars.repository.*` or similar to avoid confusing people who think it might refer to the taskgraph repo itself.